_shared/hooks/post_gen_project.py

```diff
@@ -20,7 +20,7 @@ def remove_conditional_files():
     {% endif %}
 
     {% if cookiecutter.get("postgres") != "yes" %}
-    paths_to_remove.extend(["bin/create-db"])
+    paths_to_remove.extend(["bin/create-db", "bin/make_db", "conf/alembic.ini"])
```

Comment:
Don't add the new `make_db` and `alembic.ini` files to projects that don't have DBs.